config.rb

```diff
@@ -0,0 +1,8 @@
+css_dir = "/ACM_General/home/static/home/css" 
```

Comment:
I think each watch command can only do 1-1 directories with compass.
We will have to look into this later.

Can you port this over to an issue?